### DISCUSSION TITLE
Lesson 5: General Private Key Question

### DISCUSSION THREAD

**Discussion by Daruma1800** [smartcontractkit/full-blockchain-solidity-course-py#748] [Q&A]
When inputting a private key into a env and then running the script to print that account how come the return doesn't provide the actual Private Key, it provides a completely different string? I am not getting an error just curious?

**Comment by LintonAchmad** [comment]
Share your code please?

**Comment by cromewar** [comment]
Hello @Daruma1800 make sure you are importing correctly the private key and your get_account() is returning it from brownie.config.yaml (from key) other wise it would return one from ganache, so the private keys will mismatch.

**Reply by LintonAchmad** [reply]
@Daruma1800 also make sure to add dotenv in brownie.config.yaml file

**Reply by Daruma1800** [reply]
yeah so my .env file isn't even highlighting export so I have no idea why it won't recognize the file, don't worry its a dumby account. Can you see below: 

![image](https://user-images.githubusercontent.com/94502087/150653682-5d66d05b-0aab-4037-bb43-51bc82e5acbc.png)

**Reply by cromewar** [reply]
vscode has not highly by default for env files, you need to use extension. 
did you tried using the export command directly on the terminal?
Is the problem solved now?
